packages/@tinacms/scripts/src/index.ts

```diff
@@ -123,6 +124,35 @@ export const run = async (args: { watch?: boolean; dir?: string }) => {
   }
 }
 
+const watch = () => {
```

Comment:
Same watch logic as before, but easier with `pnpm` thanks to it's `list` command.